src/Calculator/Resources/en-GB/Resources.resw

```diff
@@ -1,4 +1,4 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Added or removed a UTF8-BOM header unintentionally? If yes, please revert the change for this line.

Reply:
Yes it was unintentional.  @tian-lt

Reply:
@tian-lt How do I revert the single line?

Reply:
@tian-lt , it wasn't removed or added. It was just moved based on the addition of my code.

Reply:
> Yes it was unintentional. @tian-lt How do I revert the single line?

You can simply use the same file from the main branch to replace this one and then commit the change.

Reply:
@tian-lt fixed this.

Reply:
Looks like the irrelevant changes increased a lot.
Did you copy the `en-US` to `en-GB`? (I saw the word _Colour_ has been changed into _Color_.

Reply:
I did accidentally copy the wrong file.. my mistake. @tian-lt 
I don't know why that BOM is being added. I didn't touch that line

Reply:
No worries. I think we are almost good to finish this PR.

One last **TODO**: Revert all the changes made to this en-GB resource file. It is unnecessary to add localization strings to `Resources.resw` files except the ***en-US*** one, because we have a team dedicated to translating en-US strings to all the other languages Calculator supports.

Reply:
@tian-lt Finished